nuntium/templates/nuntium/profiles/welcome.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base_manager.html" %}
+{% load i18n %}
+
+{% block header %}
+{% include 'nuntium/profiles/top_menu.html' with section='your-instances' %}
+{% endblock header %}
+
+{% block content %}
+
+{% if messages %}
```

Comment:
This messages stuff feels like something that should be in a base template. Perhaps move it to base_manager?